### PR TITLE
Ensure JsonDB orders by path

### PR DESCRIPTION
**Pull request by jimmidyson**
Otherwise DB records were not consistently allocated to correct object...

### CODE REVIEW

**Comment by jimmidyson** [comment]
retest this please

**Comment by jimmidyson** [comment]
@chirino Happy for me to merge? This causes an issue if DB returns records not ordered by path (closes and opens items so you get multiple items with same path prefix).

File: jsondb/src/main/java/com/redhat/ipaas/jsondb/JsonDB.java [hunk 1]
```diff
@@ -135,7 +135,7 @@ default String push(String path, byte[] json) {
 
     ///////////////////////////////////////////////////////////////////
     //
-    // Methods for working with JSON as Streams.. These methods provide
+    // Methods for working with JSON as Streams.. These methods provide`
```
Comment (rhuss) [inline jsondb/src/main/java/com/redhat/ipaas/jsondb/JsonDB.java]: easter egg :-)

**Comment by jimmidyson** [comment]
Ta! [merge]

File: jsondb/src/main/java/com/redhat/ipaas/jsondb/JsonDB.java (hunk 1 above)
Comment (jimmidyson) [inline jsondb/src/main/java/com/redhat/ipaas/jsondb/JsonDB.java]: Ha yeah i tidied that up before merge :-b

**Comment by chirino** [comment]
Yep. Looks good I had assumed db would auto sort by PK.

**Pull request merged**